operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -52,14 +52,15 @@
     /**
      * @param vertx The vertx instance.
      * @param logContext A string used for context in logging.
+     * @param logState The state we are waiting for use in log messages
      * @param pollIntervalMs The poll interval in milliseconds.
      * @param timeoutMs The timeout, in milliseconds.
      * @param ready Determines when the wait is complete by returning true.
```

Comment:
I wonder if instead of `ready` we should use the term `completed` not to conflate with the idea of a CR being ready.

Just in the case of deletes it seems slightly strange to pass a deletion boolean supplier as 'ready' but it really is a case of semantics... just a thought not expecting a change unless you strongly agree.

Reply:
I think that makes very good sense. Thanks for raising it.